### PR TITLE
added jenkins/nexus integration

### PR DESCRIPTION
**Pull request by Chrislevi**
@hagzag

### CODE REVIEW

File: tasks/get-plugins.yml [hunk 1]
```diff
@@ -7,9 +9,17 @@
     owner: root
     group: root
 
+- set_fact:
+    plugin_url: "{{ cj_nexus_url }}/{{ cj_nexus_mirror }}"
+  when: cj_nexus
```
Comment (hagzag) [inline tasks/get-plugins.yml:14]: not the correct prefix in this role the variable prefix should be the role namespace ! which is juseppe_

File: tasks/get-plugins.yml [hunk 2]
```diff
@@ -7,9 +9,17 @@
...
+
 - name: "Get plugin to local cache dir"
   get_url:
-    url: "http://mirrors.jenkins-ci.org/plugins/{{ item.name }}/{{ item.version }}/{{ item.name }}.{{ item.extention | d('hpi') }}"
+    url: "{{ plugin_url |d('http://mirrors.jenkins-ci.org') }}/plugins/{{ item.name }}/{{ item.version }}/{{ item.name }}.{{ item.extention | d('hpi') }}"
```
Comment (hagzag) [inline tasks/get-plugins.yml:18]: Same as my previous review the prefix should be the role name - what is plugin_url !?

File: tasks/set-plugin-list.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    set_fact:
+      juseppe_plugins: "{{ juseppe_plugins }} + {{ cj_blueocean_plugins }}"
+    when: (cj_blueocean is defined) and (cj_blueocean | bool)
+
+  - name: "merge plugin lists -> workflow"
+    set_fact:
+      juseppe_plugins: "{{ juseppe_plugins }} + {{ cj_workflow_plugins }}"
+    when: (cj_workflow is defined) and (cj_workflow | bool)
+
+  when: (cj_wizard is not defined ) and (cj_wizard | bool)
```
Comment (hagzag) [inline tasks/set-plugin-list.yml:56]: This file looks exactly like the on in custom-jenkins role ... I can understand why here you would use the the cj_* for the plugins sets ... so I guess this one is OK

**Review by hagzag — CHANGES_REQUESTED** [review]

File: tasks/set-plugin-list.yml (hunk 3 above)
Comment (Chrislevi) [reply tasks/set-plugin-list.yml:56]: I tried to find a short resolution to merge dicts, but got into corners so I just used the one from jenkins.

File: tasks/get-plugins.yml (hunk 1 above)
Comment (Chrislevi) [reply tasks/get-plugins.yml:14]: cj_nexus_url already exist and its inherited from group_vars, functionally its working. its a code cosmetic issue

File: tasks/get-plugins.yml (hunk 2 above)
Comment (Chrislevi) [reply tasks/get-plugins.yml:18]: look at the set_fact in previous review. if cj_nexus trigger is true use the nexus url if not, use default which points to mirror.jenkins.org.

**Comment by Chrislevi** [comment]
If you want to keep this role clean from inherited vars(cj_*) than I should push those changes to a side branch like the other repos.

**Review by hagzag — APPROVED** [review]

**Pull request merged**